targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/reloc_vector_table.c

```diff
@@ -48,7 +48,7 @@
 #endif
 
 #if defined(__CC_ARM) || (defined(__ARMCC_VERSION) && (__ARMCC_VERSION >= 6010050))
-    __attribute__ ((section(".bss.nvictable"),zero_init))
+    __attribute__ ((section(".bss.nvictable")))
```

Comment:
While here I'd strip the above test down to `defined(__ARMCC_VERSION)`. That sort of stripping has already occurred in the core codebase.

(That form of the test was always weird - it means "ARMC5 or (ARMC and ARMC6)". Trying to construct a complicated test for "any version of ARMC" with a perfectly good test for "any version of ARMC" in the middle of it - __ARMCC_VERSION being defined).

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/2032bbf7f298454b295632cb32f8e4d951f4ec85..488f06b7e1b264779e6b76d29e39ca9c58bb967d).